patient-management-service/README.md

```diff
@@ -0,0 +1,100 @@
+# Patient Management Service (Mediflow) - Ballerina
+
+Sample for Patient Management Service service.
+
+### Prerequisites
+1. Fork the repositoy
+
+## Getting started
```

Comment:
Shall we go with **Deploy in Choreo**? Suggesting so that the titles are consistent across other similar samples.